manifests/v2/modules/ldap.pp

```diff
@@ -16,12 +16,12 @@
 # [*net_timeout*]
 # [*port*]
 # [*start_tls*]
-# [*cacertdir*]
-# [*certfile*]
+# [*app_pki_ca_dir*]
+# [*app_pki_ca*]
```

Comment:
This should be app_pki_cert (app_pki_ca is the CA cert)

Also it should default to $app_pki_dir  ($::<whatever module you define pki dir in but I would define at the top with  source definition.) Not /etc/pki.  Nothing is ever put automatically in /etc/pki.